docs/services/containerservice/onboarding/new_user_walkthrough.md

```diff
@@ -0,0 +1,92 @@
+# New user onboarding for the K8s GPU cluster
+
+## 1) New user sign up to EIDF
+
+All users need an account on the EIDF portal which provides users with a desktop VM to access the  GPU cluster.
+
+Full Documentation on signing up at [EIDF Documentation](https://epcced.github.io/eidf-docs/).
+
+a. Open Browser and goto the [EIDF Portal](https://portal.eidf.ac.uk)
+
+b. Click Login -> This will redirect to SAFE
+
+c. if you have a SAFE account -> use that account, if you do not have a SAFE account, register for a new SAFE account
```

Comment:
Do we want to include a link into the SAFE docs? i.e. https://epcced.github.io/safe-docs/safe-for-users/#register